test/scripts/install-kind.sh

```diff
@@ -10,11 +10,9 @@ sudo mv ./kubectl /usr/local/bin/kubectl
 # Install e2e test binary and ginkgo
 # The e2e binaries are published upstream so the ecosystem can consume then directly
 # xref https://github.com/kubernetes/enhancements/blob/master/keps/sig-testing/20190118-breaking-apart-the-kubernetes-test-tarball.md
-# Current e2e binary version for CI is kubernetes 1.20 with the fix for IPv6 network policies #96856
-# We can always get the latest version published from the URL dl.k8s.io/ci/latest.txt
-# curl -L dl.k8s.io/ci/latest.txt
-# v1.21.0-alpha.0.341+46d481b4556e33
-curl -LO https://dl.k8s.io/ci/v1.21.0-alpha.0.341+46d481b4556e33/kubernetes-test-linux-amd64.tar.gz
```

Comment:
get `v1.21.0-beta.1.341+2f8a2258f45b40`  to unblock the CI and we'll figure out the definitive solution tomorrow
This will give us 90 days until the binary expires